src/material-experimental/mdc-chips/chips.scss

```diff
@@ -1 +1,56 @@
-// TODO: MDC core styles here.
+@import '@material/chips/mixins';
+@import '../mdc-helpers/mdc-helpers';
+
+@include mdc-chip-without-ripple($query: $mat-base-styles-query);
+@include mdc-chip-set-core-styles($query: $mat-base-styles-query);
+
+
+// The MDC chip styles related to hover and focus states are intertwined with the MDC ripple styles.
+// We currently don't use the MDC ripple due to size concerns, therefore we need to add some
+// additional styles to restore these states.
+.mat-mdc-chip:not(.mat-mdc-chip-disabled):not(.mat-mdc-basic-chip) {
+  &:hover, &:focus {
+    &::after {
+      content: '';
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      background: rgba(0, 0, 0, 0.2);
+      top: 0;
+      left:0;
+      pointer-events: none;
+    }
+  }
+}
+
+// Angular Material supports disabled chips, which MDC does not.
+// Dim the disabled chips and stop MDC from changing the icon color on click.
+.mat-mdc-chip-disabled {
+  opacity: 0.4;
+
+  .mat-mdc-chip-trailing-icon {
+    pointer-events: none;
+  }
+}
+
+// Angular Material supports vertically-stacked chips, which MDC does not.
+.mat-mdc-chip-set-stacked {
+  flex-direction: column;
+  align-items: flex-start;
+
+  .mat-mdc-standard-chip {
+    width: 100%;
+  }
+}
+
+// Don't apply any styles, including MDC styles, to basic chips.
+.mat-mdc-basic-chip {
+  all: unset;
```

Comment:
`unset` is unfortunately unsupported in IE11, and the `all` property isn't supported for both IE11 and Edge. I think for now we have to continue having the basic chips be unstyled by omission of CSS classes. this change could go in a follow-up PR, though, if want just want to add a TODO now

Reply:
So previously when I had wrapper divs in the chip templates, I had something like this in the css files:

.mat-mdc-standard-chip {
  @include mdc-chip-without-ripple($query: $mat-theme-styles-query);
}

Then we moved to putting the classes all on the host component. Which I like because it cleaned up the template files a lot! But now, since the mdc classes are on the same component that has the mat-mdc-standard-chip-class, I didn't see how to write the css to conditionally include the mdc styles for only standard chips. (Nesting it like above didn't work because then the mdc styles are only applied to something matching the selector ".mat-mdc-standard-chip .mdc-chip"). So the best thing I came up with was to let mdc apply its styles to all chips, and then try to unset them for basic chips. But I see what you're saying about that not being supported!

How do people usually solve this?

Reply:
You could do it two ways:
1. Add another directive for just the styled variant that only adds the CSS classes:
```ts
@Directive({
  selector: 'mat-chip, [mat-chip]',
  host: {'class': 'mat-mdc-chip ...'}
})
export class MatChipCssInternalOnly { }
```

2. In the constructor, just check the DOM element and add the class directly:
```ts
constructor(elementRef: ElementRef, ...) {
  if (!elementRef.nativeElement.nodeName === 'mat-basic-chip' ...) {
    elementRef.nativeElement.classList.add('mat-mdc-chip', ...);
  }
}
```

Reply:
Thank you!! Those are great suggestions. I went with #1.